mindjunkies/courses/migrations/0011_alter_course_status.py

```
@@ -0,0 +1,18 @@
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('courses', '0010_remove_course_published_course_status_and_more'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='course',
+            name='status',
+            field=models.CharField(blank=True, choices=[('draft', 'Draft'), ('published', 'Published'), ('archived', 'Archived')], default='draft', max_length=10),
```

Ruff (0.8.2)

Line too long (163 > 119)

(E501)